### PR TITLE
Support delete core models, inherit excluding from Obsidian

### PR DESCRIPTION
**Pull request by Emt-lin**
related：#785、#756

### CODE REVIEW

File: src/VectorStoreManager.ts [hunk 1]
```diff
@@ -286,8 +286,21 @@ class VectorStoreManager {
   private async getFilePathsForQA(filterType: "exclusions" | "inclusions"): Promise<Set<string>> {
     const targetFiles = new Set<string>();
 
-    if (filterType === "exclusions" && this.settings.qaExclusions) {
-      const exclusions = this.settings.qaExclusions.split(",").map((item) => item.trim());
+    // not documented in Obsidian API, but I got this answer from Obsidian's discord
+    const userIgnoreFilters: string[] = (app.vault as any).getConfig("userIgnoreFilters");
```
Comment (logancyang) [inline src/VectorStoreManager.ts]: It could be better to be a bit more defensive in case `getConfig` fails.

**Comment by logancyang** [comment]
SCR-20241126-pykx

SCR-20241126-pyhn


The notice doesn't show the combined exclusions correctly.

**Comment by Emt-lin** [comment]
@logancyang Can we need to add some default exclusions.(eg: copilot-conversations、copilot-custom-prompts folders)

**Comment by logancyang** [comment]
> @logancyang Can we need to add some default exclusions.(eg: copilot-conversations、copilot-custom-prompts folders)

Good point. Custom prompts definitely needs to be excluded. Conversations may be helpful in some cases.

**Pull request merged**